### DISCUSSION TITLE
How to do Authentication Init?

### DISCUSSION THREAD

**Discussion by WebDevYCH** [aliyazdi75/flutter_chat#10] [Q&A]
Hello, i am studying flutter chat and I feel awesome on this project.
I am a beginner.  
How can I do Authentication Init to see the app working?
I believe you will help me.
Thank you.

**Comment by aliyazdi75** [comment]
Hi, this is a sample application without any server. You can read the code and use it on your own project. It is not something that could work because there is no server to respond.
